CHANGES.rst

```diff
@@ -1,3 +1,13 @@
+Version 3.1.0
+-------------
+
+Unreleased
+
+-   Remove previously deprecated code.
+-   Pass extra keyword arguments from ``get_or_404`` to ``session.get``. :issue:`1149`
+-   ``Pagination.next()`` enforces ``max_per_page``. :issue:`1201`
```

Comment:
@orlandobcrra It looks like you brought in the changes from main for 3.1.0, which shouldn't be on 3.0.x.
 
Can you 
1) merge from 3.0.x so you get 3.0.4 changes 
2) replace the top section with this instead?

```
Version 3.0.5
-------------

Unreleased

-   ``Pagination.next()`` enforces ``max_per_page``. :issue:`1201`
```